pages/_document.tsx

```diff
@@ -6,6 +6,17 @@ export default class MyDocument extends Document {
       <Html lang="en-AU">
         <Head>
           <link href="https://fonts.googleapis.com/css?family=Hind:400,500,700&display=swap" rel="stylesheet" />
+          <style>
+            {`
+svg {
```

Comment:
Specificity is low enough here that it'll be overwritten by pretty much any styles that come in later. I expect when we fix the FOUC we'll be able to remove this, it's just a quick improvement